examples/03_eval_finetuned.py

```diff
@@ -67,9 +69,11 @@ def main(_):
     env = HistoryWrapper(env, horizon=1)
     env = RHCWrapper(env, exec_horizon=50)
 
-    # wrap env to handle action/proprio normalization -- match normalization type to the one used during finetuning
-    env = UnnormalizeActionProprio(
-        env, model.dataset_statistics, normalization_type="normal"
+    policy_fn = supply_rng(
```

Comment:
Can you add a small comment here that explains that the `supply_rng` helper creates a fresh rng every time it's called